machine/v1beta1/types_gcpprovider.go

```diff
@@ -129,6 +139,13 @@ type GCPMachineProviderSpec struct {
 	// ShieldedInstanceConfig is the Shielded VM configuration for the VM
 	// +optional
 	ShieldedInstanceConfig GCPShieldedInstanceConfig `json:"shieldedInstanceConfig,omitempty"`
+
+	// ConfidentialCompute Defines whether the instance should have confidential compute enabled.
```

Comment:
Nnit, comments should start with the json name 
```suggestion
	// confidentialCompute Defines whether the instance should have confidential compute enabled.
```

Reply:
All other comments in this struct start with the field name and not the json name.

Reply:
They all need an update then 😅 We shouldn't continue against the conventions just because other fields around are breaking them, especially in this case where it's a documentation thing (I plan to enforce this in APIs later once I have the time to write the linter for it)

Reply:
NP, I'll update all of them